requirements.txt

```diff
@@ -12,6 +12,7 @@ PyYAML>=3.12
 absl-py
 kaggle
 requests
+tables
```

Comment:
why is this required?

Reply:
This is actually needed for the Dask PR, if I remember correctly, so I'll remove this for now and add it in that PR.

Reply:
not a huge deal, but moving it will likely keep things cleaner

Reply:
Fixed.